tools/ci_build/github/azure-pipelines/linux-qnn-ci-pipeline.yml

```diff
@@ -32,11 +32,11 @@ parameters:
 - name: QnnSdk
   displayName: QNN SDK version
   type: string
-  default: 2.21.0.240401
+  default: 2.22.0.240425
 
 jobs:
   - job: Build_QNN_EP
-    pool: onnxruntime-qnn-ubuntu-2004-cpu
+    pool: onnxruntime-qnn-ubuntu-2204-cpu
```

Comment:
Note: The Linux QNN pipeline is now using an Ubuntu 22.04 image. The QNN android pipeline is still using the old Ubuntu 20.04 image because I haven't been able to properly install android ndk dependencies. I'll handle that in a separate PR.